tools/browserutils/virtualenv.py

```diff
@@ -0,0 +1,49 @@
+import os
+import sys
+import logging
+from distutils.spawn import find_executable
+
+from utils import call
+
+logger = logging.getLogger(__name__)
+
+class Virtualenv(object):
+    def __init__(self, path):
+        self.path = path
+        self.virtualenv = find_executable("virtualenv")
+        if not self.virtualenv:
+            raise ValueError("virtualenv must be installed and on the PATH")
+
+    @property
+    def exists(self):
+        return os.path.isdir(self.path)
+
+    def create(self):
+        if os.path.exists(self.path):
+            shutil.rmtree(self.path)
+        call(self.virtualenv, "--no-download", self.path)
```

Comment:
The ```--no-download``` option was removed in [pip version 7.0.0](https://github.com/pypa/pip/blob/master/NEWS.rst#700-2015-05-21).